client/app/components/queries/AddToDashboardDialog.jsx

```diff
@@ -89,7 +90,9 @@ function AddToDashboardDialog({ dialog, visualization }) {
           value={searchTerm}
           onChange={event => setSearchTerm(event.target.value)}
           suffix={
-            <CloseOutlinedIcon className={searchTerm === "" ? "hidden" : null} onClick={() => setSearchTerm("")} />
+            <PlainButton className={searchTerm === "" ? "hidden" : null} onClick={() => setSearchTerm("")}>
+              <CloseOutlinedIcon />
```

Comment:
Do Icons not support click events / aria roles? Is this a styling change?

Reply:
Wrapping in a button is better than native behavior for two reasons:

1. Out of the box keyboard support for `Enter` and `space`, according to the specs, without need for additional logic.
2. Better semantics: despite being a close icon, it is, after all, a close _button_, and this makes it explicit both in the code and for screen readers.
3. Extra one: only elements with interactive roles (which I surely could have added) can have `aria-label`s

Reply:
I think I buy it! 👍